edge-modules/api-proxy-module/Cargo.lock

```diff
@@ -873,9 +945,15 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "88d6731146462ea25d9244b2ed5fd1d716d25c52e4d54aa4fb0f3c4e9854dbe2"
 dependencies = [
  "lazy_static",
- "windows-sys",
+ "windows-sys 0.36.1",
```

Comment:
It's a bit unfortunate that `windows-sys` is getting duplicated, but it seems unavoidable since we are already using the latest version of `schannel`.